### DISCUSSION TITLE
[General] Question about L1 CL node configuration after Ecotone upgrade

### DISCUSSION THREAD

**Discussion by ghost** [ethereum-optimism/developers#236] [General]
### Discussion Topic

I understand that after the Ecotone upgrade op-node would fetch blobs from L1 CL but then should I disable the `prune-blobs` flag?

https://github.com/sigp/lighthouse/pull/3852

As seen here L1 CL node could prune blobs, if the blobs are pruned could op-node operate without historic blobs?

( Asking since the recommended configuration with L1 CL nodes doesn't exist )

**Comment by trianglesphere** [comment]
If you want to keep the blobs around you can disable pruning. op-node will have snap sync & we are also working on a blob archiver to allow the CL to prune blobs while retaining the ability to sync.

**Reply by ghost** [reply]
@trianglesphere So it is in progress and at the time being when those features aren't available, we need to disable pruning for L1 CL nodes right?

**Reply by trianglesphere** [reply]
No, ecotone is not active

**Reply by ghost** [reply]
@trianglesphere You mean those features, blob archiver and snap sync will be available at the time when op mainnet enables Ecotone?

**Reply by trianglesphere** [reply]
yes. Also, the op-stack cannot use blobs until ecotone is active
